pcf/test/particle/aws/ecr/test_ecr_repository.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Capital One Services, LLC
```

Comment:
you should be using our new test generator framework. should just be a few config lines.

Reply:
Changed to use the new test generator framework